### ISSUE TITLE
There is no "SmartView&Embeddings" setting option in the settings interface.

### ISSUE THREAD

**Issue by KENT525107** [brianpetro/obsidian-smart-connections#880]
![1](https://github.com/user-attachments/assets/2afa010f-5f4b-43f8-a019-f5406d464d25)
![2](https://github.com/user-attachments/assets/bdd09ed7-dc8c-41eb-b83d-f464d73b6346)

**Comment by KENT525107** [comment]
I can't find the option for me to set up model. Both Obsidian and Smart Connection are ensured to be the latest versions. I have also tried restarting, but still there is none.

**Comment by brianpetro** [comment]
An error might be preventing the section from loading.

Check the developer console logs for errors, screenshot them and I'll give them a look.

The error should appear right when you open the settings 🌴

**Comment by KENT525107** [comment]
I am very sorry. I don't understand code. I am not sure if the screenshot I gave you is an error report of smart connection. Please understand.

**Comment by KENT525107** [comment]
![1](https://github.com/user-attachments/assets/22244d8b-73b6-4a26-b8b0-e20207f7d2ea)
![2](https://github.com/user-attachments/assets/ba42fecb-d2dc-49e8-921d-435cd853c054)
![3](https://github.com/user-attachments/assets/394681c9-9ad3-4dd0-837d-6ac9475cad02)

**Comment by KENT525107** [comment]
Can you help me solve this problem?

**Comment by brianpetro** [comment]
It looks like there is probably an erroneous exclusion in the "folder exclusions" setting, and that's preventing everything from working properly. See the first screenshot error where "/" is excluded, meaning all files in your vault are excluded 🌴

**Comment by KENT525107** [comment]
Now there is a new problem. The embedding is running normally, but there is still no option to set the model.

**Comment by KENT525107** [comment]
![q](https://github.com/user-attachments/assets/41101692-ace7-44ee-986b-88b924bd3578)
![w](https://github.com/user-attachments/assets/817a9e34-5d86-4218-8c76-a3cba3e39056)
![e](https://github.com/user-attachments/assets/ffa6e051-8c96-4cf7-ba8f-4a6a644a3c01)

**Comment by KENT525107** [comment]
![r](https://github.com/user-attachments/assets/0779ab7a-0cfe-47a4-b7ff-774c26bdaccb)

**Comment by KENT525107** [comment]
Hello, is there anything else I can do now?

**Comment by brianpetro** [comment]
@KENT525107 are things still not working as expected? Can you help me better understand how I can help you? 🌴

**Comment by KENT525107** [comment]
Yes, it still cannot be used. Only the relevance interface is working properly. However, there is still no setting that allows me to set up the model. So I don't know what to do. I'm sorry.

**Comment by KENT525107** [comment]
![20241202](https://github.com/user-attachments/assets/4a0648a7-5b4e-48f1-a081-ad21a9c0a0c1)

**Comment by KENT525107** [comment]
![202412021](https://github.com/user-attachments/assets/33250018-8731-4434-b550-44e1fd0ee5a0)
